src/compiler/parse/state/tag.ts

```diff
@@ -181,6 +182,22 @@ export default function tag(parser: Parser) {
 		element.expression = definition.value[0].expression;
 	}
 
+	if (name === 'svelte:element') {
+		const index = element.attributes.findIndex(attr => attr.type === 'Attribute' && attr.name === 'this');
+		if (index === -1) {
+			parser.error(parser_errors.missing_element_definition, start);
```

Comment:
didnt find a test case for this, can add a validator test to test if we throw the error if the `this` attribute is missing.

example: https://github.com/sveltejs/svelte/tree/master/test/validator/samples/tag-invalid

Reply:
Added test!
https://github.com/sveltejs/svelte/pull/6898/commits/31573fd1750bda98915aa91e3d5d95312dadd583

Reply:
looking good 👍🏻